packages/wonder-blocks-data/components/data.js

```diff
@@ -0,0 +1,187 @@
+// @flow
+import * as React from "react";
+import {Server} from "@khanacademy/wonder-blocks-core";
+
+import {RequestFulfillment} from "../util/request-fulfillment.js";
+import {ResponseCache} from "../util/response-cache.js";
+import {TrackerContext} from "../util/request-tracking.js";
+
+import type {Result, IRequestHandler} from "../util/types.js";
+
+type Props<TOptions, TData> = {|
+    handler: IRequestHandler<TOptions, TData>,
```

Comment:
Can you add comments for these props? Right now they show up in the documentation but have no description.

Reply:
Yes